source/ed/osm2ed.cpp

```diff
@@ -551,7 +618,7 @@ void OSMCache::fusion_ways() {
  * Ways are supposed to be order, but they're not always.
  * Also we may have to reverse way before adding them into the polygon
  */
-void OSMRelation::build_polygon(OSMCache& cache) const {
+void OSMAdminRelation::build_polygon(OSMCache& cache) {
```

Comment:
Why have you lost the constness? The function doesn't seem to have changed :cry:

Reply:
this `constness` was just a smoke screen, the method was mutating `mutable` attributes ...

Since I removed the `mutable` attributes, this method needs needs to be not const (this method builds and sets the `polygon`)